packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -198,6 +205,21 @@ export class TreeItem extends FASTElement {
         }
     };
 
+    public childItemLength(): number {
+        const treeChildren: HTMLElement[] = this.childItems.filter(
+            (item: HTMLElement) => {
+                return isTreeItemElement(item);
+            }
+        );
+        return treeChildren ? treeChildren.length : 0;
+    }
+
+    public isNestedItem(): null | boolean {
```

Comment:
Might make sense for this to be a readonly property that always returns a boolean.